demo_modules/_template/template_client.ts

```diff
@@ -0,0 +1,76 @@
+import { Client } from "../../client/modules/module_interface.ts";
+import { ModuleWS } from "../../lib/websocket.ts";
+import { Logger } from "../../lib/log.ts";
+import { Polygon } from "../../lib/math/polygon2d.ts";
+import { CanvasSurface } from "../../client/surface/canvas_surface.ts";
+import { ModulePeer } from "../../client/network/peer.ts";
+import { ModuleState } from "../../client/network/state_manager.ts";
+
+export function load(
+  // Default logging lib, with multiple levels.
+  _debug: Logger,
```

Comment:
Remove the _, because if this stands as a template, people will think you should leave them in!

Reply:
(just disable the lint errors as needed)